src/components/text-area/text-area.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ChangeEvent, useState } from 'react';
+import { ReactComponent as LockIcon } from '../../assets/svg/security-lock.svg';
+import './text-area.scss';
+
+export interface TextAreaProps {
+  lines: number;
+  initialText?: string;
+  label?: React.ReactNode;
+  placeholder?: string;
+  readonly?: boolean;
+  resizable?: boolean; // default: false
+  onChange?: (value: string) => void;
+}
+
+export const TextArea = ({
+  lines,
+  initialText,
+  label,
+  placeholder,
+  readonly,
+  resizable,
+  onChange,
+}: TextAreaProps) => {
+  const [value, setValue] = useState(initialText);
+
+  return (
+    <div className="c-text-area-wrapper">
+      {label && <label className="c-text-area-label">{label}</label>}
+      <textarea
+        className={`c-text-area ${readonly ? 'readonly' : ''}`}
+        rows={lines}
+        placeholder={placeholder}
```

Comment:
That's nice that nothing shows up when the placeholder is undefined. I'll going to simplify that in mine, haha

Reply:
Yup, as long as it's not a field that makes the component uncontrolled, `undefined` will act as if it just ignored the attribute.

Reply:
That makes it easier on our end, haha